cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -365,3 +366,61 @@ def _cleanup():
         cleanup_target(provider, migrated_vm)
 
     assert src_vm_obj.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.tier(2)
+def test_osp_if_no_password_is_exposed_in_logs_during_migration(appliance, provider, request,
+                                                                mapping_data_vm_obj_mini):
+    """
+    title: OSP: Test if no password is exposed in logs during migration
+
+    Polarion:
+        assignee: sshveta
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for Vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1. Mapping created and visible in UI
+            2.
+            3. logs should not show password during migration
+    """
+    cred = []
+    cred.append(credentials["vsphere6-nested"]["password"])
```

Comment:
We don't have to hardcode vmware provider , what if the test is run against vsphere67-ims ?
You can pass source_provider(which is vmware provider) to the test and use it .